experimental/examples/wheel/BUILD

```diff
@@ -0,0 +1,66 @@
+# Copyright 2018 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+package(default_visibility=["//visibility:public"])
+
+licenses(["notice"])  # Apache 2.0
+
+load("//python:python.bzl", "py_library", "py_test")
+load("//experimental/python:wheel.bzl", "py_wheel")
+
+py_library(name="main",
```

Comment:
In this example main.py appears to act as a binary rather than a library. Can this be changed to a `py_binary` rule, or will that change the meaning of the `py_wheel`s below? If it changes the meaning then that should be clearly documented in `py_wheel`.

Reply:
I think packaging py_binaries is wrong, for two reasons:
1) py_binary expects to be executed as script. On the other hand, entry points specify a callable to run, which means that not only the file is imported, but a specific callable within the module gets executed.

2) py_binary adds it's own shim for finding runfiles etc; on the other hand, with wheels one would expect the installer (usually pip) to create its own kind of shim.

I've added a comment in the rule definition.